releasenotes/notes/ensure-ddtracerun-respects-disabled-integrations-ea1952a79a353bf0.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    tracing: Resolves an issue where ``DD_TRACE_<INTEGRATION>_ENABLED=False`` could not be used to disable
+    the flask, django, bottle, falcon, and pyramid integrations.
```

Comment:
but only when using `ddtrace-run` ?

Reply:
Yup, I just omitted that piece because it got wordy. Should I make that distinction clear?

Reply:
personal opinion... I think it will help people understand if/how they are impacted. if they aren't using `ddtrace-run` they can ignore this note for example.